### PR TITLE
REOPEN. Adds nanotrasen pin, pride pins and pride undershirts. 

### PR DESCRIPTION
**Pull request by SynthTwo**
## What Does This PR Do

Adds pride pins and pride undershirts. This is a reopen of https://github.com/ParadiseSS13/Paradise/pull/25808. Adjusted the colours of the pins as well to be more accurate. Adds a nanotrasen pin as the main pin item, you can find it in loadout.

Was told to keep the pin overlays from everyone more important than me. Sorry Ramon babes.

## Why It's Good For The Game

In quote of the closed PR:

**"Adds a fluff item that allows for greater character expression, and was requested by several players.".**

Before you make an argument for it being lore-inaccurate, here's this quote from the PR:

**"I think the great thing about this game being completely fictional is that we can modify in whatever way we like. Especially in a direction that would validate and celebrate a very large portion of our community in a fun non-obstructive manner.**

**Sexuality/Gender identity is not something that is going to disappear into oblivion because it's 700 years into the future. I'm sure these flags would look different over time, but hey, why re-invent the wheel?"**

Worried it's too political? Another quote from the PR:

**"I do get peoples concern about this pr, however the genderless PR went fine, and TG station and goonstation have had these pins for quite a while, with no issue in that regard. As for this being a political issue, I would agree the topic can be very controversal with some. However in that regard, we have already made our stance as welcoming like most other ss13 severs, if the sever icon is any sign of it. If anything, we are doing catch up, not making an unexpected political stance no other ss13 sever has done."**

:)

## Images of changes

image 
image
image
image
image

## Testing

Spawned a bunch of mobs in game, made them wear the stuff, put on the pin, took off the pin, checked the radius menu. Standard stuff. 

## Changelog
:cl:
add: Adds pride pins to loadout preference
add: Adds nanotrasen pin to loadout preference
add: Adds pride undershirts to character creation 
/:cl:

### CODE REVIEW

**Review by lewcc — APPROVED** [review]
Code LGBT 👍

**Comment by SynthTwo** [comment]
smilecat

File: code/modules/clothing/under/accessories/accessory.dm [hunk 1]
```diff
@@ -814,6 +814,48 @@
 	item_state = "corset_blue"
 	item_color = "corset_blue"
 
+/obj/item/clothing/accessory/pride_pin
```
Comment (S34NW) [inline code/modules/clothing/under/accessories/accessory.dm:817]: Can you kindly make a main pin item with the core functionality at a higher level to allow additional pins in future? The pride pin can then be a subtype of the pin

Comment (SynthTwo) [reply code/modules/clothing/under/accessories/accessory.dm:817]: Can do! That's a great idea, people have expressed wanting lore pins.

**Comment by Cress67** [comment]
VoxPins
Finally, this image will be reproducible on Paradise. Truly, one small step for man, and one giant leap for mankind.

**Comment by AffectedArc07** [comment]
> > I think the great thing about this game being completely fictional is that we can modify in whatever way we like.
> 
> You mean the way you like. I seriously doubt anything right leaning from 2024 would be accepted into the game. Not that anyone would be snowflaky enough to try anyway.

Wait are you telling me we cant have Nigel Farrage?

bro......

**Comment by Fordoxia** [comment]
> > > I think the great thing about this game being completely fictional is that we can modify in whatever way we like.
> > 
> > 
> > You mean the way you like. I seriously doubt anything right leaning from 2024 would be accepted into the game. Not that anyone would be snowflaky enough to try anyway.
> 
> Wait are you telling me we cant have Nigel Farrage?
> 
> bro......

No Farrage, why even live?

**Review by DGamerL — APPROVED** [review]

**Comment by Scribble-Sheep** [comment]
I absolutely love that the pride pins start off as an NT pin, lmao.
Like "you may have your pride, but first you have to purchase our logo," it's so perfectly NT in a way I can't explain lol

**Comment by ZER0-In** [comment]
Bruh

File: code/modules/mob/new_player/sprite_accessories/sprite_accessories.dm [hunk 2]
```diff
@@ -433,6 +433,30 @@
...
+	icon_state = "shirt_pansexual"
+
+/datum/sprite_accessory/undershirt/shirt_asexual
+	name = "Asexual Shirt"
+	icon_state = "shirt_asexual"
+
+/datum/sprite_accessory/undershirt/shirt_rainbow
+	name = "Rainbow Shirt"
+	icon_state = "shirt_rainbow"
 //end graphic shirts
```
Comment (DGamerL) [inline code/modules/mob/new_player/sprite_accessories/sprite_accessories.dm:460]:
```suggestion
	icon_state = "shirt_rainbow"

//end graphic shirts
```

**Pull request merged**